docs/userguide/definemonitor.rst

```diff
@@ -46,6 +46,10 @@ Linked Monitors
     This field allows you to select other monitors on your system that act as triggers for this monitor. So if you have a camera covering one aspect of your property you can force all cameras to record while that camera detects motion or other events. You can either directly enter a comma separated list of monitor ids or click on ‘Select’ to choose a selection. Be very careful not to create circular dependencies with this feature however you will have infinitely persisting alarms which is almost certainly not what you want! To unlink monitors you can ctrl-click. 
 
 Maximum FPS 
+
+    .. warning::
+      Unless you know what you are doing, please leave this field empty, especially if you are configuring a network camera. More often than not, putting a value here adversely affects live viewing performance.
+
```

Comment:
I would say affects recording performance as opposed to live viewing.  The actual recordings will be junk as ffmpeg is starved for cpu and can't keep up with receiving the stream.

Reply:
resolved by https://github.com/pliablepixels/ZoneMinder/commit/723e973e8805b2cef1b012c578c5fd3f45ec7518